fearless/Modules/Crowdloan/CrowdloanContributionConfirm/CustomFlows/MoonbeamContributionConfirmInteractor.swift

```diff
@@ -109,14 +109,29 @@ class MoonbeamContributionConfirmInteractor: CrowdloanContributionConfirmInterac
             if let ethereumAccountAddress = self?.ethereumAccountAddress,
                let memoCall = self?.makeMemoCall(memo: ethereumAccountAddress) {
                 nextBuilder = try nextBuilder.adding(call: memoCall)
-
-                self?.saveEthereumAdressAsMoonbeamDefault()
```

Comment:
Why we shouldn't save it here?

Reply:
it was a mistake to put it here. we have a such logic inside app: if user sends referral address once - don't put this address in future extrinsics because it fee-costable operation. by having save referral address here, we have a such case: if add_memo will be failed by some reason, app will not include this memo in future calls. so I replaced this saving to the place where we ensure that transaction was succeed